### PR TITLE
Add blog for introduction to c# and playwright

### PR DESCRIPTION
**Pull request by bimalghartimagar**

### CODE REVIEW

File: 2023/09/introduction-to-playwright-in-c#.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Automating web application tasks is one of the necessary skills for software developers and testers. You need it for performing repetitive tasks or conducting end-to-end testing of web applications and sometimes scraping data from websites. In this blog post, we'll explore how to use C# with Playwright for automating tasks.
+
+### What is Playwright?
+
+Playwright is an open-source automation framework developed by Microsoft that allows you to automate web applications using various programming languages, including C#. Playwright was created specifically to accommodate the needs of end-to-end testing, however we will use it as a library for web automation. Playwright supports all modern rendering engines including Chromium, WebKit, and Firefox. Test on Windows, Linux, and macOS, locally or on CI, headless or headed with native mobile emulation.
+
+To begin web automation using C# and Playwright, follow these steps:
+
+1. **Prerequisites:** You should have .NET Core or .NET 5+ installed on our machine.
```
Comment (dylanwooters) [inline 2023/09/introduction-to-playwright-in-c#.md:18]: Change to "your machine"?

File: 2023/09/introduction-to-playwright-in-c#.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      await browser.CloseAsync();
+      ```
+
+**Advantages of Using C# and Playwright**
+
+1. **Cross-Browser Support:** Playwright supports all modern rendering engines including Chromium, WebKit, and Firefox.
+
+3. **Cross-Platform Support:** Test on Windows, Linux, and macOS, locally or on CI, headless or headed.
+
+2. **C# Language Familiarity:** Playwright API can be used with C#. Having familiarity with C# and usign it for web automation can reduce the learnig curve.
```
Comment (dylanwooters) [inline 2023/09/introduction-to-playwright-in-c#.md:89]: "usign" to "using". "learnig" to "learning"

File: 2023/09/introduction-to-playwright-in-c#.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+4. **Headless and Headful Browsing:** Playwright allows you to choose between headless and headed mode. It makes it easier to write tests and debug an issue in headed mode as we can see what's going on in every line of code.
+
+5. **Powerful APIs:** With Playwright's APIs, you can perform complex web automation tasks with ease.
+
+6. **Powerful Tooling:** It has powerful tooling like [Codegen](https://playwright.dev/dotnet/docs/codegen), [Playwright inspector](https://playwright.dev/dotnet/docs/debug#playwright-inspector) and [Trace Viewer](https://playwright.dev/dotnet/docs/trace-viewer-intro). They help in making the playwright usage easier 
+
+7. **Emulation:** You can test your app on any browser or emulate a real device such as mobile phone or tablet. It can emulate behavior such as ``userAgent``, ``geolocation``, ``viewport`` and many  more. You can read more about it in the [emulation](https://playwright.dev/dotnet/docs/emulation) documentation
+
+There are many other features in playwright which are surely useful for any use cases.
```
Comment (dylanwooters) [inline 2023/09/introduction-to-playwright-in-c#.md:101]: "any" to "many"?

**Review by dylanwooters — COMMENTED** [review]
Great post! I really liked using Playwright after you suggested it, so happy to see a post on it. I made a few comments just around grammatical stuff.

**Comment by bimalghartimagar** [comment]
@dylanwooters thank you for reviewing the blog post. Indeed, playwright is a nice and emerging tool. I have fixed the issues and pushed again.

**Review by dylanwooters — APPROVED** [review]
Looks good!

**Comment by sethjensen1** [comment]
@bimalghartimagar This is live. Thanks!

**Pull request closed without merging**